homework.md

```diff
@@ -4,83 +4,84 @@
 
 What is 2 + 2?
 
-5
+4.
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+White.
 
 ## Question 3
 
 What is the  2 * 3 * 6 * 1 * 10 * 34?
 
-1322
+12240.
 
 ## Question 4 
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
+Hypertext Markup Language.
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+Describes the presentation of a webpage or document.
 
 ## Question 6
 
 Using CSS, how would I select every `<h1>` tag on the page?
 
 ```css
-#h1 {
-
+h1 {
 }
 ```
 
 ## Question 7
 
 What is JavaScript?
 
-A programming language used on the internet
+A programming language that can make webpages interactive.
```

Comment:
On the contrary JavaScript is more than just a programming language for web interactivity and answering the question the way you did, is for another question let me explain myself.

So JavaScript ever since 2009 when NodeJS was introduced one could now build **web services and CLI's** with JavaScript, come 2015 and React Native was introduced and you could now create Native Mobile Applications with JavaScript and finally 2019 came ElectronJS one can now build Desktop Applications with JavaScript... 

# So my point here is JavaScript is more than what you say it is and adding "Web interactivity" you basically answering "what JavaScript does" not "what JavaScript is".

Reply:
Ah, I will see to that and do more research on JavaScript, thanks for the input again Ntshembo.